### PR TITLE
Rename uPickle => weePickle

### PR DESCRIPTION
**Pull request by htmldoug**
As suggested in https://github.com/rallyhealth/weePickle/pull/9#pullrequestreview-294642608, it'd be useful to have a unique name to avoid confusion with lihaoyi/upickle.

Naming credit: @michaelleahy.

### CODE REVIEW

File: bench/src-js/Main.scala [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-import com.rallyhealth.upickle.v1.ADTs.ADT0
-import com.rallyhealth.upickle.v1.Common.{Data, bench}
-import com.rallyhealth.upickle.v1.Defaults.ADTc
-import com.rallyhealth.upickle.v1.Hierarchy.{A, B, C}
-import com.rallyhealth.upickle.v1.Recursive.{End, LL, Node}
+import com.rallyhealth.weepickle.v1.ADTs.ADT0
+import com.rallyhealth.weepickle.v1.Common.{Data, bench}
+import com.rallyhealth.weepickle.v1.Defaults.ADTc
+import com.rallyhealth.weepickle.v1.Hierarchy.{A, B, C}
```
Comment (michaelleahy) [inline bench/src-js/Main.scala]: I just wanted to try out the multi line comment...

File: readme.md [hunk 2]
```diff
@@ -8,29 +11,29 @@ Play requires upgrading much of our ecosystem, which is a hassle. We want
 to decouple our libraries from Play as much as possible, to reduce this
 friction.
 
-So we are encouraging libraries to make use of uPickle instead: it's
+So we are encouraging libraries to make use of weePickle instead: it's
 popular, well-supported and fast.
```
Comment (michaelleahy) [inline readme.md]: Is this statement necessarily true anymore?

Comment (htmldoug) [reply readme.md]: Yep, that's the goal. Thoughts?

Comment (michaelleahy) [reply readme.md]: I meant specifically that i was "well-supported".  Which isn't entirely wrong, but now that we've made it a clear fork, is that true?  I don't think its necessarily fair to count on your continued love to keep it going in perpetuity.

Comment (htmldoug) [reply readme.md]: It's a philosophical question that depends on your point of view.

It'd expect weePickle to be less widely used than lihaoyi/upickle, so we'll likely be submitting the majority of patches. Anyone is free to do that--even users in the community which is more than we can say for our internal libs.

If we need something from upstream lihaoyi/upickle, lihaoyi is really helpful in gitter.im and is leading the way with updates. I've already backported his 5-10% perf improvements from 8 days ago: https://github.com/lihaoyi/upickle/pull/284. He's cool with the fork existing, although we obviously wouldn't use him as L1 tech support for it.

In many ways, the fork better supports our needs. Things like Scala 2.11, play 2.5 support and behavior changes like https://github.com/rallyhealth/weePickle/pull/4 to ease our migration path aren't reasonable to expect "the community" to support for us. You can also compare it to play-json 2.5, which went EOL in Sep 2018.

**Pull request merged**